performance-tests/bench_2/test_controller/ScenarioManager.cpp

```diff
@@ -308,9 +360,9 @@ std::vector<WorkerReport> ScenarioManager::execute(const AllocatedScenario& allo
   const std::chrono::seconds timeout(allocated_scenario.timeout);
   std::shared_ptr<std::thread> timeout_thread;
   if (timeout.count() > 0) {
-    timeout_thread.reset(new std::thread([&reports_left, &reports_left_mutex, &timeout_cv, &timeout, &guard_condition]  {
+    timeout_thread.reset(new std::thread([&] {
```

Comment:
I think I had a reason for capturing them by name, but I can't remember it know. After some googling it looks like it should have the same result, so it might have been feeling I had to be explicit, so this is fine.